distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.url.mvn.cfg

```diff
@@ -36,7 +36,7 @@
 #   * 3. if not found looks for ${maven.home}/conf/settings.xml
 #   * 4. if not found looks for ${M2_HOME}/conf/settings.xml
 #
-#org.ops4j.pax.url.mvn.settings=
+org.ops4j.pax.url.mvn.settings=thisdirectorydoesnotexist/settings.xml
```

Comment:
I understand the intent in the context of this PR, but perhaps we should add a comment as to why `thisdirectorydoesnotexist` needs to be there.